tests/test_vpc.rb

```diff
@@ -56,247 +49,251 @@ def test_vpc_create
 
   def test_vpc_destroy
     # create and test again
-    @vpc = Vpc.new(100)
-    @vpc.destroy
+    vpc = Vpc.new(100)
+    vpc.destroy
     refute(Vpc.enabled, 'VPC feature should have been disabled')
```

Comment:
disabling the feature may be needed to clear the configuration easily. Also, doesn't the resource 'ensure = absent' rely on the destroy method to clear the config?